docs/core/tools/dotnet-workload-list.md

```diff
@@ -44,8 +41,6 @@ For more information about the `dotnet workload` commands, see the [dotnet workl
 
 [!INCLUDE [no-cache](../../../includes/cli-no-cache.md)]
```

Comment:
Should this and some of the others be removed?

Reply:
Yes, good catch, thanks!